docs/content/Workspace/Inspecting-Pre-aggregations.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Pre-aggregation inspection
+permalink: /cloud/inspecting-pre-aggregations
+category: Workspace
+menuOrder: 5
+---
+
+The Pre-Aggregations page in Cube Cloud allows you to inspect all
+pre-aggregations in your deployment at a glance. You can see which
+pre-aggregations are accelerating queries, which are not, and which are not
+being refreshed, along with the last 24 hours of build history.
```

Comment:
Should we have links from "pre-aggregations" and maybe "refreshed" so the Cube Cloud features are interlinked with relevant Cube Core features?